src/Engine/ProtoCore/DSASM/InstructionSet.cs

```diff
@@ -734,6 +734,7 @@ public bool TryGetArrayKey(out StackValue array, out int index)
         /// <returns></returns>
         public StackValue ToBoolean(Core core)
         {
+            RuntimeCore runtimeCore = core.__TempCoreHostForRefactoring;
```

Comment:
Can we adding a property `RuntimeMemory` to core, otherwise it is so tedious to write
```
RuntimeCore runtimeCore = core._tempcoreHostforRefactoring;
RuntimeMemory runtimeMemory = runtimeCore.RuntimeMemory;
```